ansible-st2/roles/arteria_node/tasks/arteria_runfolder.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+#TODO Ensure all this is versioned!
+
+- name: get arteria-core from github for runfolder
```

Comment:
If we want "stand alone" installations of the services to depend on Github, then yes, I agree.